source_pubsub/oauth.go

```diff
@@ -112,19 +144,25 @@ func UserInfo(oauthEndpoint, accessToken string) (*UserInfoResponse, error) {
 
 	defer httpResp.Body.Close()
 
-	var userInfoResponse UserInfoResponse
-	err = json.NewDecoder(httpResp.Body).Decode(&userInfoResponse)
+	var userInfoResp UserInfoResponse
+	err = json.NewDecoder(httpResp.Body).Decode(&userInfoResp)
 	if err != nil {
 		return nil, fmt.Errorf("error decoding user info - %s", err)
 	}
 
 	if httpResp.StatusCode != http.StatusOK {
-		return nil, fmt.Errorf("non-200 status code returned on OAuth user info call: code=%v, error=%v, error_description=%v",
-			httpResp.StatusCode,
-			userInfoResponse.Error,
-			userInfoResponse.ErrorDescription,
+		return nil, fmt.Errorf(
+			"unexpected oauth userInfo (status: %d) response: %w", httpResp.StatusCode, userInfoResp.Err(),
```

Comment:
does this include the error description? in the past, we had a lack of visibility in the error, and relied on the description for the actual error

Reply:
yup, this is a helper. https://github.com/conduitio-labs/conduit-connector-salesforce/pull/107/files/a208b30291728a20abe8e2e5d8aa0cde2301be4f#diff-b6e396f22e8f40ce1b32cde2e5eb223e9abce57888fbd57df6a5d1394547b4f2R63